### PR TITLE
fix: Represent "True" as JSON boolean instead

### PR DESCRIPTION
**Pull request by confiks**
This is non-substantive change.

***
<a href="https://pr-preview.s3.amazonaws.com/confiks/vc-data-model/pull/730.html" title="Last updated on Jun 14, 2021, 2:23 PM UTC (5dc1b37)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/730/46fedab...confiks:5dc1b37.html" title="Last updated on Jun 14, 2021, 2:23 PM UTC (5dc1b37)">Diff</a>

### CODE REVIEW

**Comment by David-Chadwick** [comment]
Yes I agree with this change

**Comment by OR13** [comment]
can we please merge this?

**Review by OR13 — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Comment by confiks** [comment]
Just as a heads-up: the contribution is non-substantive and thus no copyright applies.

**Comment by TallTed** [comment]
It seems that the comment from @iherman at https://github.com/w3c/vc-data-model/pull/737#issuecomment-734082004 applies here as well (i.e., errata process should be followed).

**Comment by msporny** [comment]
Waiting on process suggestion to be approved by @iherman before merging.

**Review by msporny — APPROVED** [review]

**Comment by brentzundel** [comment]
@ivan, this is a change to some example text, so I do not believe it represents a problem as far as IPR is concerned.

**Review by brentzundel — APPROVED** [review]

**Review by clehner — APPROVED** [review]

**Comment by confiks** [comment]
Can be marked by someone with the appropriate rights as non-substantive at https://labs.w3.org/repo-manager/pr/id/w3c/vc-data-model/730

**Comment by msporny** [comment]
hrm @clehner -- typically, only Chairs, Staff Contact, or Editors are allowed to make the "non-substantive" determination because there are IPR implications for doing so. In this case, it was fine, but in almost every other case, we wait on @iherman to make the call. :)

**Comment by brentzundel** [comment]
Yes, seconding Manu's comment:  the chairs are responsible for determining whether there are IPR concerns, and we rely on our staff contact to remove the flag.
tl;dr - really great initiative, but next time better not :)

**Review by peacekeeper — APPROVED** [review]

**Comment by clehner** [comment]
@msporny @brentzundel @iherman 
Apologies; I will be more observant.
It could be unmarked so that Staff Contact could mark it correctly.

**Comment by kdenhartog** [comment]
I'll include this PR in the round of PRs that I'll send to the CCG mailing list over the next day or two to get appropriate review on this per our review process.

**Review by kdenhartog — APPROVED** [review]

**Comment by kdenhartog** [comment]
multiple positive reviews, no objections, merging
